torch/_dynamo/symbolic_convert.py

```diff
@@ -1462,13 +1505,51 @@ def RESUME(self, inst):
         pass
 
     def BINARY_OP(self, inst):
-        if sys.version_info >= (3, 11):
+        # linter complains about dis missing _nb_ops in py < 3.11
```

Comment:
nit: since you revert the condition from the previous commit here, the diff is a bit weird and this comment you removed is back again.